.gitignore

```diff
@@ -16,4 +16,11 @@
 *.out
 *.app
 example
-./shell
+/vic
```

Comment:
@vargac well, didn't you mean `./vic` then? I am not exactly sure but `/vic` matches, well, `/vic`.

Did it actually work? Could you see `vic` binary in `git status`?